src/django_otp_webauthn/backends.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any
+from django.contrib.auth import get_user_model
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.http import HttpRequest
+
+from django_otp_webauthn.models import AbstractWebAuthnCredential
+
+UserModel = get_user_model()
+
+class WebAuthnBackend:
+    """A simple authentication backend used when django_otp_webauthn is used for passwordless authentication"""
+
+    def authenticate(self, request: HttpRequest, web_authn_credential: AbstractWebAuthnCredential | None = None, **kwargs: Any) -> AbstractBaseUser | None:
+        if web_authn_credential:
+            user = web_authn_credential.user
+            return user if self.user_can_authenticate(user) else None
+        return None
+    
+    def get_user(self, user_id) -> AbstractBaseUser | None:
+        user = UserModel._default_manager.get(pk=user_id)
+        return user if self.user_can_authenticate(user) else None
+    
+    def user_can_authenticate(self, user: AbstractBaseUser | None) -> bool:
+        """
+        Reject users with is_active=False. Custom user models that don't have
+        that attribute are allowed.
+        """
+        return bool(user and getattr(user, "is_active", True))
```

Comment:
This is taken straight from Django's `ModelBackend`, although I did add the check for `user` being truthy which is not in Django's code. I added that because initially `get_user()` was using `UserModel._default_manager.filter(pk=user_id).first()` so as to not blow up and then just always passing that into `self.user_can_authenticate()`. I reverted back to just handling `get_user()` the same way `ModelBackend` does, but left the handling of a falsey (and probable `None` in that case) user being passed in since Python won't prevent that from happening and returnin a `False` seems valid for that case.